publisher/publisher.go

```diff
@@ -109,6 +110,9 @@ func main() {
 	if conf.awsRegion == "" {
 		l.Fatal("missing 'aws_region' value")
 	}
+	if conf.runID == "" {
```

Comment:
It is possible to cover this use case by test?

Reply:
Current implementation is highly procedural and there're no tests for main func.
I think we should address this within https://github.com/newrelic/infrastructure-publish-action/issues/26